packages/design-system/src/components/Table/TableCell.tsx

```diff
@@ -72,7 +72,6 @@ export const TableCell: React.FC<
   id,
   scope,
   stackedTitle,
-  stackedClassName,
```

Comment:
Is this a bug fix? Did this need to be applied to the cell element and wasn't until now?

Reply:
Linting flagged this because it was declared and not used in the component. I didn't see it being used in the component or elsewhere in the repo so I assumed it was safe to remove.

Reply:
By removing it from this destructuring assignment, we're actually _adding_ it to the `tableCellProps` object, so this will now be applied to the cell element. I think you found a bug if it's not being used. We should probably remove this from this PR, though, because it does change behavior.

Reply:
Ohhhh OK